gnomad_tgps.py

```diff
@@ -0,0 +1,86 @@
+import hail as hl
```

Comment:
since this is also only getting run once, it might be helpful to document here what this was for, something like:
```
"""
Script to check the number of common (AF > 1%) variants per gnomAD sample.

Eimear Kenny (one of the co-chairs of the Human Pangenome Reference Consortium; HPRC) requested that we compute the number of common variants in v3 samples that aren't present in the pangenome reference samples. They want to use these counts (grouped by genetic ancestry group) to assess the new pangenome reference.

Additional context from Eimear:
Karen Miga (cc’d) and I are co-chairing the population sampling and representation (PSR) working group in the NHGRI-funded [Human Pangenome Reference Consortium](https://secure-web.cisco.com/1wEtWOLTKVYtWJnbArjS4zghgKUFX93o1sN0mQVOivAUXbXPCfCHkfPLVkVm2tKIBKn8F09kdp7Y2J7RsLvEGAZCDs8W0yTMKoGWm1s0Y7Veo2yh4KTEVjOjb8TLy3gaV4YVZGSbzg-6YVKJ_1zCluG0nu1giv_57B7R3iOj-KDT8Mbs33S7WnnMVRKX38hFbprm_rLF5k9zrzPnubWnilml5K-LXjYDbMWvh_-tf8bxtJNMxo6D5CcgIUH6lbCw-rZ7H5uE4N8QET9QlOYh-2XWhlZLK3lP6wSUFzwKTTtVvfHZ7xJlxT4OxjcvzeKlg/https%3A%2F%2Furldefense.proofpoint.com%2Fv2%2Furl%3Fu%3Dhttps-3A__secure-2Dweb.cisco.com_1Ob8k7wHNA5C-5FvKQ-2DMidxIlMcXRAW6e8DYvyiZJlTukKHPaP8fq82TJBcWA-2DjN4jaYePv-5FG1L7gsqVpEWL6pS7T126E2o-2Dn3-5FSYwkF8liXrI6QfwMp8gnV2iI4pnh07A5Il9tJVR2E0nXNU7LQu5QdjfvvER4qtpNgkg47-2DE9yImA2Y4eTNK33rDDoxeAErt9VA1cNsumCKr-5FwisGOC0ucFuSN9XAJmT7Qg23xBbqwVu9fSHIEopyhMi-5Ft7eRxj69v8YacRWRAUotxXCQqBSQ00zlQNXF5-2Dl-2DBzIY1c5Hr9eiD3fKIKftt7puHSg5vhW0_https-253A-252F-252Fhumanpangenome.org-252F%26d%3DDwMGaQ%26c%3DshNJtf5dKgNcPZ6Yh64b-ALLUrcfR-4CCQkZVKC8w3o%26r%3DIt_OrWdcyK0f9wuF6wEGfIU7GYNY2rxI2d4LiAPqQog%26m%3D24G11qOM7WQAORgzDsMwEKlyTHXyirbcsMar9EfDM5SV33qYFhOfIdqqyRo0s_gL%26s%3D3KF5qL_zWekMU32XRAckan0Yh1LyjSsXXumojECpvtE%26e%3D) (HRPC). The goal of the consortium is to update the current human genome reference by producing a pangenome of high quality long read sequencing of 350 diverse humans which will become the new pangenome reference. The PSR group is responsible to selecting samples to be included in the pangenome, which should comprehensively cover most common variants, defined as variants >1%, in human populations globally. We are halfway through (phase 1) of the project, and most of the sampling has been selected from the 1000 Genome Cell lines (N=174 samples). We have developed a simple variant counting metric that helps us evaluate anticipated performance of the pangenome using short read data in phase one samples and additional out-of-sample genomes from the 1KG project. Here is a link to a brief slide deck that describes this:
 
[https://www.dropbox.com/s/fo5nrxerxwukw7x/20220908.HPRC-PopulationSampling.Gnomad.pdf?dl=0](https://secure-web.cisco.com/1iVOw0lVHiaU92htBwgWMiX_OxYzmOWE-iAb3GMr4NVU3vDZ_bA3EWlI_mi4kkPKVrfUZIDj1XpNHNCx2rvj_DXOcCyFCPtNACVbK81VLUp0QC9urt0CmYs5hzNYvQRNZJDzR4DuzjHGTuKJ6W20Vr5zzpxIEtdoABGDCdlf7A6n3Z9P8l9B6gZ5ey5BeJfOASXbC4ygmlqalywPIPGAQGhZ2qz27kNWLSwzmSQszJpW5oWgAl_RKyFXoQ_xyO1OtjkFUVO2Ihy1MlY-jVXMDgJG03EeIRWVnoerYFFgtOgj75mshXjmSUHjyTdSaRw8Z/https%3A%2F%2Furldefense.proofpoint.com%2Fv2%2Furl%3Fu%3Dhttps-3A__secure-2Dweb.cisco.com_1nL2RMyl1NSUaHNovJ9XR62WInYM-2DqLmpwrAKYZHQ0ehKPwwFGK7KTrXUb-5FPGL5uGcjYEcayx-2Dh0LYtGsXwdB-5FYTBCzI9r3zhU3ryc3k4346tfABfHIiAdiN6p2Qh5smySAugCxgVnD2bafSbZq0dG1wESu8zCjACUGcog5FwLa1IoEWCDuBwHuYbv3ic2KtwGiOvTaighCcmdPSObmwPVztX3WBYQCx7GvNSM4CaEiaeMjBKqyMEznBEGj2VIml3zsrlvMpIzvCXXspeyivqygNwH9aeTq5SHdCz1GILZP7jgg6s2UyvqHJHJLxJVWi3_https-253A-252F-252Fwww.dropbox.com-252Fs-252Ffo5nrxerxwukw7x-252F20220908.HPRC-2DPopulationSampling.Gnomad.pdf-253Fdl-253D0%26d%3DDwMGaQ%26c%3DshNJtf5dKgNcPZ6Yh64b-ALLUrcfR-4CCQkZVKC8w3o%26r%3DIt_OrWdcyK0f9wuF6wEGfIU7GYNY2rxI2d4LiAPqQog%26m%3D24G11qOM7WQAORgzDsMwEKlyTHXyirbcsMar9EfDM5SV33qYFhOfIdqqyRo0s_gL%26s%3DIrDl2DBerjCYiEHylOca5CJAN1Ks6v7d8axY6sPsvj0%26e%3D)
 
We would love to additionally be able to assess the performance in Gnomad – not only is this an independently ascertained dataset, but it would also give us a look ahead to how aligned the pangenome will be to the top resource used by clinical testing labs. Ideally, we would like to run the analysis on individual-level data in Gnomad, and I’m reaching out to see if you would be willing to collaborate with us on this analysis. The script is a very simple variant count algorithm, so hopefully very straightforward to run. We are currently writing up this work, and of course, anybody who participates in this analysis would be included as a co-author.
"""
```